doc/wiki/Installation:-Option-List.md

```diff
@@ -0,0 +1,147 @@
+|:---:|:---:|:---:|---|---|---|
+| `--elbdm_scheme`     | `ELBDM_WAVE`, `ELBDM_HYBRID`                    | `ELBDM_WAVE`    | ELBDM schemes. `ELBDM_WAVE`: wave-only; `ELBDM_HYBRID`: fluid-wave-hybrid-scheme | Must set `--model=ELBDM` | <a name="--elbdm_scheme"></a> `ELBDM_SCHEME` |
+| `--wave_scheme`      | `WAVE_FD`, `WAVE_GRAMFE`                        | `WAVE_FD`       | Wave schemes. `WAVE_FD`: finite difference; `WAVE_GRAMFE`: local spectral method | Must set `--model=ELBDM` | <a name="--wave_scheme"></a> `WAVE_SCHEME` |
+| `--conserve_mass`    | `true`, `false`                                 | `true`          | Enforce the mass conservation | Must set `--model=ELBDM` | <a name="--conserve_mass"></a> `CONSERVE_MASS` |
+| `--laplacian_four`   | `true`, `false`                                 | Depend          | Enable the fourth-order Laplacian | Must set `--model=ELBDM` and `--wave_scheme=WAVE_FD` | <a name="--laplacian_four"></a> `LAPLACIAN_4TH` |
+| `--gramfe_scheme`    | `GRAMFE_MATMUL`, `GRAMFE_FFT`                   | `GRAMFE_MATMUL` | GramFE schemes. `GRAMFE_MATMUL`: faster for `PATCH_SIZE`=8; `GRAMFE_FFT`: faster for larger patch sizes | Must set `--model=ELBDM` and adopt `--wave_scheme=WAVE_GRAMFE`. Must set `--fftw` for `--gramfe_scheme=GRAMFE_FFT` and `--gpu=false` | <a name="--gramfe_scheme"></a> `GRAMFE_SCHEME` |
```

Comment:
- "`GRAMFE_MATMUL`: matrix-multiplication approach, which is faster for `PATCH_SIZE`=8; `GRAMFE_FFT`: FFT approach, which is faster for larger patch sizes"
- "Must set `--fftw` when using both `--gramfe_scheme=GRAMFE_FFT` and `--gpu=false` | <a name="--gramfe_scheme"></a> `GRAMFE_SCHEME`"